src/shared/lib/StoreUtils.ts

```diff
@@ -900,10 +901,33 @@ export async function queryOncoKbData(
                   body: structuralQueryVariants,
               });
 
+    const indicatorMapResult = generateIdToIndicatorMap(
+        mutationQueryResult.concat(structuralVariantQueryResult)
+    );
+
+    const set = new Set<string>();
+    for (const key in indicatorMapResult) {
+        if (indicatorMapResult[key].highestSensitiveLevel !== null) {
+            set.add(ONCOKB_LEVEL_TYPE_ENUM.TXS);
+        }
+        if (indicatorMapResult[key].highestResistanceLevel !== null) {
+            set.add(ONCOKB_LEVEL_TYPE_ENUM.TXR);
+        }
+        if (
+            indicatorMapResult[key].highestPrognosticImplicationLevel !== null
+        ) {
+            set.add(ONCOKB_LEVEL_TYPE_ENUM.PX);
+        }
+        if (
+            indicatorMapResult[key].highestDiagnosticImplicationLevel !== null
+        ) {
+            set.add(ONCOKB_LEVEL_TYPE_ENUM.DX);
+        }
+    }
+
     const oncoKbData: IOncoKbData = {
-        indicatorMap: generateIdToIndicatorMap(
-            mutationQueryResult.concat(structuralVariantQueryResult)
-        ),
+        indicatorMap: indicatorMapResult,
+        levelTypeSet: set,
```

Comment:
Also here. I think this is for calculating the space? And it's not working yet?

Reply:
It works now